### PR TITLE
remark header.S

### PR DESCRIPTION
**Pull request by AnanSN**

### CODE REVIEW

**Comment by AnanSN** [comment]
first merge

File: arch/x86/boot/header.S [hunk 1]
```diff
@@ -12,7 +12,8 @@
  * addresses must be multiplied by 16 to obtain their respective linear
  * addresses. To avoid confusion, linear addresses are written using leading
  * hex while segment addresses are written as segment:offset.
- *
+ * 
+ * 在实模式使用64k段,所以,段地址 * 16 = 线性地址
```
Comment (songjiayang) [inline arch/x86/boot/header.S]: don't use Chinese , change to English please.

**Comment by daxtens** [comment]
There is a documentation section in Chinese - maybe that would be an
appropriate place for this contribution?
https://github.com/torvalds/linux/tree/master/Documentation/translations/zh_CN

File: arch/x86/boot/header.S (hunk 1 above)
Comment (AnanSN) [reply arch/x86/boot/header.S]: I will close it.

**Pull request closed without merging**